umbral/utils.py

```diff
@@ -41,4 +41,6 @@ def kdf(ecpoint, key_length):
 
 
 def get_curve_keysize_bytes(curve):
+    # We use the ceil operation to fit all bytes on curve sizes where eight is
+    # not evenly divisible.
     return int(math.ceil(curve.key_size / 8.00))
```

Comment:
I think it's more common to see this expressed as `(curve.key_size + 7) // 8`

See e.g., https://github.com/pyca/cryptography/blob/master/src/cryptography/utils.py#L66

Reply:
It also seems to be much faster:
```
>>> timeit.repeat('for i in range(1024): (i+7)//8', number=10000, repeat=3)
[2.6502825429779477, 2.350924202008173, 2.525679681042675]
>>> timeit.repeat('for i in range(1024): int(math.ceil(i/8.00))', number=10000, setup='import math')
[6.333626834035385, 6.332635344995651, 6.0212114860187285]
```

Reply:
Very nice, thanks!